db/seeds.d/75-job_templates.rb

```diff
@@ -1,7 +1,9 @@
 if Katello.with_remote_execution?
   User.as_anonymous_admin do
     JobTemplate.without_auditing do
-      Dir[File.join("#{Katello::Engine.root}/app/views/foreman/job_templates/**/*.erb")].each do |template|
+      template_files = Dir[File.join("#{Katello::Engine.root}/app/views/foreman/job_templates/**/*.erb")]
+      template_files.select! { |file| !(file =~ /\_ansible\_default\.erb$/) } unless Katello.with_ansible?
```

Comment:
Do we need an regex here, wouldn't

```ruby
template_files.select! { |file| !file.end_with?('_ansible_default.erb') } unless Katello.with_ansible?
```

suffice?

Reply:
Any reason for not using `#reject` instead of `#select` with negated condition?